src/EnergyPlus/VariableSpeedCoils.cc

```diff
@@ -7127,9 +7124,6 @@ namespace VariableSpeedCoils {
         //+ VarSpeedCoil(DXCoilNum)%BasinHeaterPower + VarSpeedCoil(DXCoilNum)%EvapCondPumpElecPower
         state.dataVariableSpeedCoils->QSource *= PartLoadRatio;
 
-        //  Add power to global variable so power can be summed by parent object
-        state.dataHVACGlobal->DXElecCoolingPower = state.dataVariableSpeedCoils->Winput;
-
```

Comment:
Moved `state.dataHVACGlobal->DXElecCoolingPower` to `UpdateVarSpeedCoil()` function such the latest iteration value is used.